tests/conv/run_convs.sh

```diff
@@ -101,378 +101,380 @@ fi
 # ./layer_example iters inpWidth inpHeight nImg nIfm nOfm kw kh padw padh stride type
 #
 if [ "f32" == "${BIN}" ]; then
-  PREC_BF16=0
+  PREC=4
```

Comment:
see comment on using a string "F32, BF16, BF8" in the driver as this prepares better for HF8